modules/nf-core/abricate/run/main.nf

```diff
@@ -2,7 +2,7 @@ process ABRICATE_RUN {
     tag "$meta.id"
     label 'process_medium'
 
-    conda (params.enable_conda ? "bioconda::abricate=1.0.1" : null)
+    conda "bioconda::abricate=1.0.1"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
         'https://depot.galaxyproject.org/singularity/abricate%3A1.0.1--ha8f3691_1':
```

Comment:
Why does this have the URL encoding `%3A` instead of `:`.

Reply:
Excellent spot 👀 Not sure but it has happened before and we should probably look for it and remove in any affected modules.